account_invoice_transmit_method/models/partner.py

```diff
@@ -0,0 +1,30 @@
+# © 2017 Akretion (http://www.akretion.com)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    customer_invoice_transmit_method_id = fields.Many2one(
+        'transmit.method', string='Customer Invoice Transmission Method',
+        company_dependant=True, track_visibility='onchange',
```

Comment:
There's a typo, it should be `company_dependent`. This typo is also present in previous versions of the module.

Reply:
good catch! I'm taking care of creating PR to fix that in `8.0` and `10.0`.

Reply:
Ping me when you open the PRs, I will quickly approve them.

Reply:
https://github.com/OCA/account-invoicing/pull/424 and https://github.com/OCA/account-invoicing/pull/425